packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -195,16 +204,31 @@ export class DashboardModel {
                 `${DashboardsTableName}.dashboard_id`,
                 `${DashboardVersionsTableName}.dashboard_id`,
             )
+            .leftJoin(
+                DashboardViewsTableName,
+                `${DashboardVersionsTableName}.dashboard_version_id`,
+                `${DashboardViewsTableName}.dashboard_version_id`,
+            )
             .select<GetDashboardQuery[]>([
                 `${DashboardsTableName}.dashboard_id`,
                 `${DashboardsTableName}.dashboard_uuid`,
                 `${DashboardsTableName}.name`,
                 `${DashboardsTableName}.description`,
                 `${DashboardVersionsTableName}.dashboard_version_id`,
                 `${DashboardVersionsTableName}.created_at`,
+                `${DashboardViewsTableName}.filters`,
             ])
             .where('dashboard_uuid', dashboardUuid)
-            .orderBy(`${DashboardVersionsTableName}.created_at`, 'desc')
+            .orderBy([
+                {
+                    column: `${DashboardVersionsTableName}.created_at`,
+                    order: 'desc',
+                },
+                {
+                    column: `${DashboardViewsTableName}.created_at`,
+                    order: 'desc',
+                },
+            ])
             .limit(1);
```

Comment:
Do you still want the limit if the dashboard can have many views?

Reply:
I'm only adding support for 1 view in this PR so the limit of 1 still makes sense. Once we support multiple views, we will have to make a separate query to fetch them.
More in this comment.
https://github.com/lightdash/lightdash/pull/1391#issuecomment-1041525876